### PR TITLE
Set persistent storage for rabbitmq and redis services

### PR DESCRIPTION
**Pull request by ArseniiPetrovich**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,12 +2,25 @@ version: "3"
 services:
   redis:
     image: "redis"
```
Comment (akolotov) [inline docker-compose.yml]: Should we specify exact version of `redis` here? Similarly as it was done in https://github.com/poanetwork/bridge-nodejs/pull/88/files#diff-ac440e3f7ab9ccaac9aeab32ac91b8b3?

File: docker-compose.yml [hunk 2]
```diff
@@ -2,12 +2,25 @@ version: "3"
...
+    command: ["redis-server", "--appendonly", "yes"]
+    hostname: redis
+    restart: on-failure
+    volumes:
+      - ~/bridge_data/redis:/data
```
Comment (akolotov) [inline docker-compose.yml]: What component is responsible for creation `~/bridge_data` on the host system?

File: docker-compose.yml (hunk 1 above)
Comment (ArseniiPetrovich) [reply docker-compose.yml]: Yes, it is definitely a good idea to explicitly specify a version of `redis` image.

File: docker-compose.yml (hunk 2 above)
Comment (ArseniiPetrovich) [reply docker-compose.yml]: @akolotov Docker creates all the folder tree automatically while building images with attached drives. Of course, user that build images should have sufficient permissions.

**Pull request merged**